share/spack/spackos-bootstrap/stage4/spack.in.yaml

```diff
@@ -0,0 +1,75 @@
+# This is a Spack Environment file.
+#
+# It describes a set of packages to be installed, along with
+# configuration settings.
+spack:
+  include:
+    - compilers.yaml
+  # add package specs to the `specs` list
+  specs:
+  # glibc from stage3, must match the gcc built there or we get two mixed
+  # together, use hash
+  - glibc @GLIBC_HASH@
+  - binutils +gold +lto
+  - gcc@13.2.0 +binutils +piclibs ~bootstrap languages=c,c++,fortran,lto
+  # base system
+  - m4
+  - ncurses
+  - bash
+  - coreutils
+  - diffutils
+  - file
+  - findutils
+  - gawk
+  - grep
+  - bzip2
+  - gzip
+  - gmake
+  - patch
+  - sed
+  - tar
+  - xz
+  - gettext
+  - bison
+  - perl
+  - vim +python
+  - nano
+  - which
+  # if building outside of spackos, python somehow grabs something out of the
+  # local include paths, build like this:
+  # $ bwrap --dev-bind / / --tmpfs /usr/local --tmpfs /usr/include/x86_64-linux-gnu spack install -v python
+  - python
+  - texinfo
+  - util-linux
+  # spack bootstrap, base requirements
+  - unzip
+  - git
+  # spack bootstrap doesn't work, install manually
+  - gnupg
+  - patchelf
+  - clingo
+  - clingo-bootstrap
```

Comment:
```suggestion
  - clingo-bootstrap +optimized
```

Reply:
Only seemed to happen on aarch64 of the two I'm testing.